.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,5 +1,5 @@
 blank_issues_enabled: false
 contact_links:
-  - name: K-9 Mail Forum
-    url: https://forum.k9mail.app/
+  - name: Mozilla Support Forum (SUMO)
+    url: https://support.mozilla.org/en-US/products/thunderbird-android
```

Comment:
I believe if we change the link to `https://support.mozilla.org/products/thunderbird-android`, users will be redirected to the appropriate translation.